framework/DataObjects/Factory.py

```diff
@@ -26,7 +26,8 @@
 
 ################################################################################
 from utils import utils
-from DataObjects.Data import Data
+from DataObjects.DataObject import DataObject as Data
```

Comment:
I imagine this is imported as Data in order to avoid a bunch of name changes throughout the file. Would it be better long term to just make all those changes? A search and replace would make very quick work of it.

Reply:
All references already removed; so I trivially removed the alias now.  Fixed.